string name
string instrument_name
float32 x
float32 y
